changelogs/unreleased/5516-hshamji-small.md

```diff
@@ -0,0 +1 @@
+Enable HTTPProxy Fractional Mirroring
```

Comment:
Adding a few more details to the changelog:
```suggestion
HTTPProxy: support fractional mirroring to a service by specifying `mirror: true` and a `weight` of 1-100. See the [traffic mirroring docs](https://projectcontour.io/docs/1.26/config/request-routing/#traffic-mirroring) for more information.
```

Reply:
Updated. Thanks!